elements/pfe-sass/maps/_typography.scss

```diff
@@ -40,45 +48,90 @@ $pf-type-sizing: (
    FontSize--xs:   pf-font-prem($pf-global--FontSize--xs),  // 12px
 );
 
-/// General component usage such as accordion headers and CTAs. Use with calc()
-/// @example scss - variable example
-///   var(--pf-c-FontSize)
-///
-$pf-type-sizing--component: (
-  FontSize:                 #{pfe-var(FontSize--md )},  // 16px
-);
+/// List of typographical HTML elements
+/// @see {mixin} pfe-content 
+$pf-content-set: (h1, h2, h3, h4, h5, h6, p, small);
 
-/// Block level typography for HTML elements
-/// Example: --pf-c-content--h3--FontSize 
+/// Typography for block level HTML elements such as heading, paragraph, and small tags
+/// @example - css 
+/// font-size: var(--pf-c-content--h1--FontSize);
+/// font-size: var(--pf-c-content--h2--FontSize);
+/// font-size: var(--pf-c-content--h3--FontSize);
+/// font-size: var(--pf-c-content--h4--FontSize);
+/// font-size: var(--pf-c-content--h5--FontSize);
+/// font-size: var(--pf-c-content--h6--FontSize);
+/// font-size: var(--pf-c-content--FontSize);
+/// font-size: var(--pf-c-content--small--FontSize);
 $pf-type-sizing--content: (
-  content--h1--FontSize:    #{pfe-var(FontSize--3xl )}, // 28px
-  content--h2--FontSize:    #{pfe-var(FontSize--2xl )}, // 26px
-  content--h3--FontSize:    #{pfe-var(FontSize--xl )},  // 20px 
-  content--h4--FontSize:    #{pfe-var(FontSize--lg )},  // 18px
-  content--h5--FontSize:    #{pfe-var(FontSize--md )},  // 16px
-  content--h6--FontSize:    #{pfe-var(FontSize--sm )},  // 14px
+  content--h1--FontSize:    #{pfe-var(FontSize--3xl)}, // 28px
+  content--h2--FontSize:    #{pfe-var(FontSize--2xl)}, // 26px
+  content--h3--FontSize:    #{pfe-var(FontSize--xl)},  // 20px 
+  content--h4--FontSize:    #{pfe-var(FontSize--lg)},  // 18px
+  content--h5--FontSize:    #{pfe-var(FontSize--md)},  // 16px
+  content--h6--FontSize:    #{pfe-var(FontSize--sm)},  // 14px
+  content--FontSize:        #{pfe-var(FontSize--md)},  // 16px  this is for standard paragraph tags
+  content--small--FontSize: #{pfe-var(FontSize--sm)},  // 14px  this is for the small tag
+);
+
+/// Sets of Titles: Mega
+/// @see {mixin} pfe-typography 
+$pf-title-set--mega: ("6xl", "5xl");
```

Comment:
```suggestion
$pf-title-set--mega: (6xl, 5xl);
```

Reply:
When these are removed, it kicks out errors for each:
```
@patternfly/pfe-styles: WARNING: 4xl was not found in $pf-title-set--mega, $pf-title-set--regular, or $pf-title-set--tiny.
@patternfly/pfe-styles:          on line 258 of ../pfe-sass/mixins/_copy-mixins.scss, in mixin `pfe-typography`
@patternfly/pfe-styles:          from line 43 of ../pfe-sass/extends/_typography_extends.scss
@patternfly/pfe-styles:          from line 5 of ../pfe-sass/__extends.scss
@patternfly/pfe-styles:          from line 8 of ../pfe-sass/pfe-sass.scss
@patternfly/pfe-styles:          from line 2 of src/pfe-vars.scss
```